src/api/services/quotesService.js

```
@@ -58,6 +78,42 @@ const getQuote = async (quoteObj) => {
+    }
+    
+    if(!isCustomQuote) {
+      apiResponse = await requestApi(url);
+    }
+
+    // Get array of authors in alphabetical order and without duplicates
+    const authors = [...new Set(apiResponse.map(quote => quote.author).sort())];
+
+    return authors;
+  } catch (error) {
+    throw error;
```

Biome

[error] The catch clause that only rethrows the original error is redundant.

These unnecessary catch clauses can be confusing. It is recommended to remove them.

(lint/complexity/noUselessCatch)